time map_load_time
float32 resolution
float32 time_resolution
uint32 depth
uint32 width
uint32 height
geometry_msgs/Pose origin